src/cleanfigure.m

```diff
@@ -641,7 +641,7 @@ function movePointsCloser(meta, handle)
 
       % If a point is part of two segments, that cross the border, we need to
       % insert a NaN to prevent an additional line segment
-      [~, ~, id_conflict] = intersect(id_first (~isInfinite_first), ...
+      [trash, trash, id_conflict] = intersect(id_first (~isInfinite_first), ...
```

Comment:
Nothing, that's just an old-style way to do `[~, ~, ...] = ...`. In old MATLAB (and probably Octave) versions, the tilde as don't-care output is not supported.

Putting `trash` or `dummy` in there, makes sure that the code might also run in older versions and makes it clear those variables are not used anywhere.

I, however, think that we might also need to place a `%#ok` at the end to suppress warnings by `mlint` (I haven't checked, so I might be wrong).